---
layout: home
title: Hi, I'm Hanxi 
subtitle: Statistical/ML Method Developer. Translator between Machine Learing and Biology. <br>
use-site-title: true
---

<br> I am a PhD candidate in the Joint CMU-Pitt PhD Program in Computational Biology, working in the <a href="https://www.immunology.pitt.edu/research/center-systems-immunology"> Center for Systems Immunolog, </a> advised by <a href="https://www.jishnulab.org/">Jishnu Das</a>. <br>
My research interest lie in interpretable machine learning, statistical modeling, and their applications in high-dimentionsal noisy sparse tabular data and large-scale high-dimensional sequence data.<br>
More specifically, my projects includes interpretable latent variable methods and interpretable attention mechanism. <br> <br>


<hr style="height:2px;border-width:0;color:gray;background-color:gray">

<b>Highlighted Publications and Projects</b> (scroll down for more):<br><br>

<ul style="height: 300px; overflow: auto">
	<li> <strong>SLIDE: Significant Latent Factor Interaction Discovery and Exploration across biological domains </strong><br>  
		Javad Rahimikollu ^, <strong>Hanxi Xiao ^</strong>,  et al <br>
		<strong>Nature Method</strong>,   ^ represent equal contribution <br>
		<a href="https://www.nature.com/articles/s41592-024-02175-z">Link to Manuscript</a> <br>
		<a href="https://www.nature.com/articles/s41592-024-02176-y">Link to Research Briefing</a> <br>
		<a href="https://www.csb.pitt.edu/2024/07/19/cpcb-students-publish-innovative-machine-learning-model-in-nature-methods/">Link to Media Coverage</a> <br>
		<a href="https://news.cornell.edu/stories/2024/03/statistical-machine-learning-can-find-unknown-factors-behind-disease">Link to Media Coverage</a> <br>
	</li><br>

</ul>



<hr style="height:2px;border-width:0;color:gray;background-color:gray">

<br>

<hr style="height:2px;border-width:0;color:gray;background-color:gray">

<b>Updates & News</b> (scroll down for more):<br><br>

<ul style="height: 300px; overflow: auto">
	<li><i>September 2024:</i> I presented my project, titled '<b>Interpretable machine learning uncovers spatial-microenvironment-specific drivers of pathogenesis in myocardial infarction</b>' as one of the three chosen doctoral student/postdoc talks at the Immunology Retreat.
	</li><br>

	<li><i>September 2024:</i> My co-authored paper, <a href="https://www.science.org/doi/10.1126/scitranslmed.adk7832">Deep humoral profiling coupled to interpretable machine learning unveils diagnostic markers and pathophysiology of schistosomiasis</a> published in Science Translational Medicine.
	</li><br>

	<li><i>August 2024:</i> My co-authored paper, <a href="https://www.nature.com/articles/s41467-024-51780-1">Aberrant mitochondrial DNA synthesis in macrophages exacerbates inflammation and atherosclerosis</a>, where I applied machine learning methods to analyze murine single-cell RNA-sequencing data is published in Nature Communications.
	</li><br>

	<li><i>August 2024:</i> My co-authored paper, <a href="https://journals.aai.org/jimmunol/article/213/6/779/267054/Regulatory-T-Cell-Insufficiency-in-Autoimmune">Regulatory T Cell Insufficiency in Autoimmune Diabetes Is Driven by Selective Loss of Neuropilin-1 on Intraislet Regulatory T Cells</a>, is published in Nature Communications.
	</li><br>

	<li><i>May 2024:</i> <a href="https://www.csb.pitt.edu/2024/07/19/cpcb-students-publish-innovative-machine-learning-model-in-nature-methods/">University of Pittsburgh</a> and <a href="https://news.cornell.edu/stories/2024/03/statistical-machine-learning-can-find-unknown-factors-behind-disease">Cornell University</a> write about my work SLIDE.	
	</li><br>

	<li><i>May 2024:</i> My co-first-authored paper where we developed a novel interpretable machine learning model, <a href="https://www.nature.com/articles/s41592-024-02175-z">SLIDE</a>, has been published at Nature Methods</a>.
	</li><br>

	<li><i>December 2023:</i> I have been awarded the Quantitative Methodologies Pilot Program (QuMP) grant as a co-principle investigator for the project "<b>Uncovering immunomodulatory spatial microniches in asthma involving Th2 and Tfh2 cells using interpretable machine learning </b>."
	</li><br>

	<li><i>December 2023:</i> My firt-authored review, <a href="https://www.tandfonline.com/doi/full/10.1080/21645515.2023.2282803">From bench to bedside via bytes: Multi-omic immunoprofiling and integration using machine learning and network approaches</a>, is published in Human Vaccines and Immunotherapeutics.
	</li><br>


</ul>

<hr style="height:2px;border-width:0;color:gray;background-color:gray">

<br>